gremlin-python/src/main/jython/gremlin_python/driver/remote_connection.py

```diff
@@ -42,7 +42,7 @@ def traversal_source(self):
 
     @abc.abstractmethod
     def submit(self, bytecode):
-        print "sending " + bytecode + " to GremlinServer..."
+        print("sending " + bytecode + " to GremlinServer...")
```

Comment:
Is use of `print()` "good"? in Java we wouldn't `System.out.println()` we would use a logging framework of some sort (i.e. log4j). Is python similar in that respect? no big deal for this PR - just asking for future reference.